<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="imageOneTwo">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bilde 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seamlessOne">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sy sammen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imageOneOne">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bilde 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="glattingOverskrift">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Sømløs kloning</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>241</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;En bit av bilde 2 limes inn i&lt;br/&gt;bilde 1, og forsøker å gjøre&lt;br/&gt;dette så sømløst som mulig.&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Velg mellom tre forhåndsvalgte&lt;br/&gt;bildekombinasjoner til høyre.&lt;br/&gt;Se originalversjonen av bilde 1&lt;br/&gt;og bilde 2. Se den ferdige &lt;br/&gt;versjonen ved å klikke på&lt;br/&gt;&amp;quot;sy sammen&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seamlessCode">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vis kode</string>
   </property>
  </widget>
  <widget class="QLabel" name="seamlessImg">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>391</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../hdr-bilder/Tree/Tree_00032.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="imageTwoTwo">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bilde 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imageTwoOne">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bilde 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seamlessTwo">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>270</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sy sammen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imageThreeTwo">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bilde 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imageThreeOne">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>320</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bilde 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seamlessThree">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>400</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sy sammen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
